integrations/kube-agent-updater/cmd/teleport-kube-agent-updater/main.go

```diff
@@ -61,12 +56,22 @@ func main() {
 	var metricsAddr string
 	var probeAddr string
 	var syncPeriod time.Duration
+	var baseImageName string
+	var versionServer string
+	var versionChannel string
+	var insecureNoVerify bool
+	var disableLeaderElection bool
 
 	flag.StringVar(&agentName, "agent-name", "", "The name of the agent that should be updated. This is mandatory.")
 	flag.StringVar(&agentNamespace, "agent-namespace", "", "The namespace of the agent that should be updated. This is mandatory.")
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "healthz-addr", ":8081", "The address the probe endpoint binds to.")
 	flag.DurationVar(&syncPeriod, "sync-period", 10*time.Hour, "Operator sync period (format: https://pkg.go.dev/time#ParseDuration)")
+	flag.BoolVar(&insecureNoVerify, "insecure-no-verify-image", false, "Disable image signature verification.")
+	flag.BoolVar(&disableLeaderElection, "disable-leader-election", false, "Disable leader election, used to run the kube-agent-updater out of Kubernetes.")
+	flag.StringVar(&versionServer, "version-server", "https://update.gravitational.io/v1/", "URL of the HTTP server advertising target version and critical maintenances.")
+	flag.StringVar(&versionChannel, "version-channel", "cloud/stable", "Version channel to get updates from.")
+	flag.StringVar(&baseImageName, "base-image", "public.ecr.aws/gravitational/teleport", "Image reference containing registry and repository.")
```

Comment:
@marcoandredinis @hugoShaka In Teleport Discover UI we'll need to replace the image with `teleport-ent`.

Reply:
I would prefer to align what we are doing with the `teleport-cluster` chart and have an `enterprise: true/false` value, even if this means we need to do more work on the chart side to build the image name.

Reply:
I think we already set those flags since https://github.com/gravitational/teleport/pull/21971

Reply:
Perfect, I'll make sure the chart properly honours them :)